incubator/hnc/pkg/reconcilers/hnc_config.go

```diff
@@ -162,6 +162,29 @@ func (r *ConfigReconciler) syncObjectReconcilers(ctx context.Context, inst *api.
 		}
 	}
 
+	// If a type exists in the forest but not exists in the Spec, we will
```

Comment:
Can we start breaking this function up into multiple ones? It's getting a bit long now.

Maybe something like:
```
func syncObjectReconcilers(...) {
  r.Forest.Lock()
  defer r.Forest.Unlock()
  syncActiveReconcilers(...)
  syncRemovedReconcilers(...)
}
```

Reply:
Thanks for the suggestion! PTAL.